srcpkgs/PolyMC/update

```diff
@@ -0,0 +1,2 @@
+site=https://github.com/PolyMC/PolyMC/tags
+pattern='tag/\K[\d]+\.[\d]+(\.[\d]+)?'
```

Comment:
update files are usually not necessary for github. why does this need one?

Reply:
Probably a relic from the time we (PolyMC) didn't use GH releases (everything prior to 1.1.0).

Reply:
It picked up a 1.2 from somewhere in the release notes, so I limited it to the tags page.

Reply:
is there a difference between the distfile listed in the template and the github-generated tag archive? If so, using that could remove the need for this

Reply:
I assume the release archive contains the submodules

Reply:
The release archive contains a shallow, recursive checkout of the corresponding tag. I contributed the workflow that creates it to make packaging, especially for void, easier.